.github/workflows/README.md

```diff
@@ -0,0 +1,39 @@
+# up-rust workflow setup documentation
```

Comment:
IMHO we do not need this line as the file name alreay suggests that contains documentation and the fact that we renamed the repository multiple times already also makes it questionable if it is a good idea to add the repo name here again ;-)